### PR TITLE
tony's exercise

### PR DESCRIPTION
**Pull request by tonytanaka**
please review my code

### CODE REVIEW

File: tonytanaka/index.js [hunk 1]
```diff
@@ -5,11 +5,15 @@ let x = Math.floor(Math.random() * count) + 1;
...
+//let sy = "%&    § !* @= ";
+let sy = " %&§!*@= ";
+//let cleanSy = sy.replace(/\s/g, "");
 /* \s is the regex for "whitespace", and g is the 
 "global" flag, meaning match ALL \s (whitespaces).*/
+let cleanSy = sy.trim();
+console.log(cleanSy);
 let countSy = cleanSy.length;
+console.log(countSy);
 let y = Math.floor(Math.random() * countSy) + 1;
```
Comment (ciamac-da) [inline tonytanaka/index.js]: Hey Tony, in line 17 you can delete +1 because when i try to run the code I get undefined after every 4-5 tries.

**Review by ciamac-da — CHANGES_REQUESTED** [review]

File: tonytanaka/index.js (hunk 1 above)
Comment (tonytanaka) [reply tonytanaka/index.js]: you`re right! 7 is the lenght, but the index goes up to 6

Comment (ciamac-da) [reply tonytanaka/index.js]: 👍🏻

**Review by ciamac-da — APPROVED** [review]
👍🏻

**Pull request merged**